SPEC.md

```diff
@@ -67,7 +68,7 @@ It will then look for this executable in a list of predefined directories. Once
 - `CNI_COMMAND`: indicates the desired operation; either `ADD` or `DEL`
 - `CNI_CONTAINERID`: Container ID
 - `CNI_NETNS`: Path to network namespace file
-- `CNI_IFNAME`: Interface name to set up
+- `CNI_IFNAME`: Interface name to set up; plugin must honor this interface name or return an error
```

Comment:
The issue I'm about to raise is not caused by your change, but it's not clear which of the interfaces this applies to. This confusion bugs me and I'm tempted to suggest to remove the CNI_IFNAME variable completely.